docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,752 @@
+express         403/180     1:00:00
+short*          401/178  1-00:00:00
+long             224/47  5-00:00:00
+large           376/172     6:00:00
+gpu               41/17     8:00:00
+multigpu          41/17  1-00:00:00
+lowpriority     118/102  1-00:00:00
+reservation     617/402 100-00:00:0
+ai-jumpstart       2/15  2-00:00:00
+allshouse           5/7    infinite
+bansil             15/4 30-00:00:00
+ce-mri             3/10 30-00:00:00
+chen               0/12 30-00:00:00
+ctbp               0/20 30-00:00:00
+.
+.
+.
+:::
+
+View information about a specific partition (e.g., `short`, `gpu`, `long`):
+
+:::{code} bash
+sinfo -p <partition_name>
+:::
+
+Or, only view nodes in a certain state:
+
+:::{code} bash
+sinfo -p <partition> -t <state>
+:::
+
+You can use the `--Format` flag to get more information or a specific format for the output:
+
+:::{code} bash
+sinfo -p <partition> -t idle --Format=gres,nodes
+:::
+
+Below command will show detailed information about all nodes in the cluster, including the node name, state, CPU architecture, memory, and available features:
+
+:::{code} bash
+sinfo -N -l
+:::
+
+View what features a node has:
+
+:::{code} bash
+sinfo -n <node> --Format=nodes,nodelist,statecompact,features
+:::
+
+View what nodes are in what state in a partition using `statecompact`:
+
+:::{code} bash
+sinfo -p <partition> --Format=time,nodes,statecompact,features,memory,cpus,nodelist
+:::
+
+## Advanced Usage
+Advanced usage of Slurm involves working with multi-node jobs, GPU jobs, and understanding priority and Quality of Service (QoS) parameters. It also involves memory management and the use of environment variables in job scripts.
+
+### Multi-node Jobs
+Multi-node jobs involve executing a single job across multiple nodes. Such jobs are typically used for computationally intensive tasks that require significant parallelization.
+
+#### Use-cases: Multi-node Jobs
+Multi-node jobs are used in scenarios where tasks can be broken down into sub-tasks that can be executed in parallel, such as simulation and modeling, machine learning training, or big data analysis.
+
+#### Code Example for Multi-node Job Submission
+:::{code} bash
+#!/bin/bash
+#SBATCH -J MultiNodeJob     # Job name
+#SBATCH -N 4                # Number of nodes
+#SBATCH -n 16               # Number of tasks
+
+# Your program/command here
+srun ./my_program
+:::
+
+### GPU Jobs
+Slurm can also manage GPU resources, allowing you to specify GPU requirements in your job scripts.
+
+#### Use-cases: GPUs
+GPU jobs are used in scenarios where tasks are parallelized and can benefit from the high computational capabilities of GPUs, such as machine learning and deep learning workloads, image processing, or simulations.
+
+#### Code Example for GPU Job Submission
+:::{code} bash
+#!/bin/bash
+#SBATCH -J GPUJob            # Job name
+#SBATCH -N 1                 # Number of nodes
+#SBATCH -n 1                 # Number of tasks
+# SBATCH --gres=gpu:1        # Number of GPUs
```

Comment:
Should be #SBATCH
and should also include 
#SBATCH -p gpu

Reply:
Nice!